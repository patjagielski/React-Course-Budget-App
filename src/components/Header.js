import React from 'react';
import ReactDOM from 'react-dom';
import {NavLink} from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Budget Application</h1>
        <NavLink to='/' activeClassName='is-active' exact={true}>Home</NavLink>
        <br />
        <NavLink to='/create' activeClassName='is-active'>Create Expense</NavLink>
        <br />
    </header>
);

export default Header;



// <NavLink to='/help' activeClassName='is-active'>Help Page</NavLink>