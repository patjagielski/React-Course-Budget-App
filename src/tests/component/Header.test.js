import { shallow } from'enzyme';
import React from 'react';
import Header from '../../components/Header';

test('render the Header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    
    // expect(wrapper.find('h1').text()).toBe('Budget Application');
    
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});