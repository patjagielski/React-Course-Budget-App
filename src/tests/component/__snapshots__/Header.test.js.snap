// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the Header correctly 1`] = `
<header>
  <h1>
    Budget Application
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <br />
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
  <br />
</header>
`;
